TACC-Cloud/hazmapper
# Task/wg 403 create asset geometry component

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [WG-403](https://tacc-main.atlassian.net/browse/WG-403)

## Summary of Changes: ##
- Adds Asset Geometry component to mimic the behavior in angular

## Testing Steps: ##
1. Set your local env to production
2. Go to [test_project_testen Map](http://localhost:4200/project/5f5d6024-4440-4099-b379-3e443ca242ff)
3. Click through the various types of Geometries (Point, Geometry Collection, Polygon, MultiPolygon etc) and make sure they appear like  [test_project_testen Prod Map](https://hazmapper.tacc.utexas.edu/hazmapper/project/5f5d6024-4440-4099-b379-3e443ca242ff)
4. Test other maps with images, video, point clouds etc and check that the Geometry metadata appears as expected and compare with prod

## UI Photos:
#### Polygon:
<img width="1530" alt="Screenshot 2024-12-17 at 9 12 37 PM" src="https://github.com/user-attachments/assets/4fc75ce7-29a4-49ce-a1c5-5f5df3b617a1" />

#### Line String
<img width="1527" alt="Screenshot 2024-12-17 at 9 13 15 PM" src="https://github.com/user-attachments/assets/1004bf88-7ed4-45fb-8d5b-cc9ff5b7036d" />

#### Point LOL at the pic
<img width="1533" alt="Screenshot 2024-12-17 at 9 13 53 PM" src="https://github.com/user-attachments/assets/4f88842a-93eb-402c-8907-ad06873bdd95" />

#### Geometry Collection

<img width="1529" alt="Screenshot 2024-12-17 at 9 14 28 PM" src="https://github.com/user-attachments/assets/37c904ff-fd86-4b7b-bfc6-7d6a18738db0" />

## Notes: ##
